### PR TITLE
Move service files to repo

### PR DESCRIPTION
**Pull request by PriyangaRamasamy**
This commit moves vpd service files to openpower-vpd-parser repository and also install the service files in systemd unit directory via meson.

Test works as expected.

Change-Id: If0ee5f3b0cc4cc949f584d3f1cb234f31a9e0dc5

### CODE REVIEW

**Review by GiridhariKrishna — COMMENTED** [review]
Not sure, but shouldn't the folder be service/file or services instead?

**Comment by PriyangaRamasamy** [comment]
> Not sure, but shouldn't the folder be service/file or services instead?

In the story, it has mentioned to have service_files/

**Review by SunnySrivastava1984 — CHANGES_REQUESTED** [review]
Have a folder named OEM at the root level and under that have these OEM specific folders/files. Also, please confirm that folder name should have "_" or "-" as I see some folder names with "-".

File: meson.build [hunk 1]
```diff
@@ -166,31 +166,20 @@ systemd_system_unit_dir = dependency('systemd').get_variable(
...
 
-install_data(
-    'ibm_vpd/system-vpd.service',
-    install_dir: systemd_system_unit_dir)
+rules = ['service_files/70-ibm-vpd-parser.rules']
 
-install_data(
-    'ibm_vpd/ibm-vpd-parser@.service',
-    install_dir: systemd_system_unit_dir)
+services = ['service_files/system-vpd.service', 'service_files/ibm-vpd-parser@.service', 'service_files/com.ibm.VPD.Manager.service', 'service_files/wait-vpd-parsers.service', 'service_files/ibm-isdimm-vpd-parser@.service', 'service_files/ibm-spi-vpd-parser@.service']
```
Comment (jinuthomas) [inline meson.build:171]: could you indent this please

File: meson.build [hunk 2]
```diff
@@ -166,31 +166,20 @@ systemd_system_unit_dir = dependency('systemd').get_variable(
 udev_dir = dependency('udev').get_variable(
         'udev_dir')
 
-install_data(
-    'ibm_vpd/system-vpd.service',
-    install_dir: systemd_system_unit_dir)
+rules = ['service_files/70-ibm-vpd-parser.rules']
```
Comment (jinuthomas) [inline meson.build:169]: could you have this in a rules folder

File: meson.build [hunk 3]
```diff
@@ -166,31 +166,20 @@ systemd_system_unit_dir = dependency('systemd').get_variable(
...
 
-install_data(
-    'ibm_vpd/com.ibm.VPD.Manager.service',
-    install_dir: systemd_system_unit_dir)
+scripts = ['scripts/wait-vpd-parsers.sh']
 
-install_data(
-    'ibm_vpd/wait-vpd-parsers.service',
-    install_dir: systemd_system_unit_dir)
+install_data(rules, install_dir: udev_dir / 'rules.d')
```
Comment (jinuthomas) [inline meson.build:175]: move these install files just after the place where the files are defined

**Review by jinuthomas — COMMENTED** [review]
overall looks good, just minor changes

**Comment by PriyangaRamasamy** [comment]
> Have a folder named OEM at the root level and under that have these OEM specific folders/files. Also, please confirm that folder name should have "_" or "-" as I see some folder names with "-".

you mean, oem/service_files ; oem/rules ; oem/scripts ?

File: meson.build (hunk 1 above)
Comment (PriyangaRamasamy) [reply meson.build]: done

File: meson.build (hunk 2 above)
Comment (PriyangaRamasamy) [reply meson.build]: done

File: meson.build (hunk 3 above)
Comment (PriyangaRamasamy) [reply meson.build]: done

**Review by jinuthomas — APPROVED** [review]
Looks good to me

**Comment by jinuthomas** [comment]
> > Have a folder named OEM at the root level and under that have these OEM specific folders/files. Also, please confirm that folder name should have "_" or "-" as I see some folder names with "-".
> 
> I would go with '-', as I see other directories in - in this repo.

The coding guidelines has it as '_'

**Comment by jinuthomas** [comment]
jenkins -> run tests please

**Comment by PriyangaRamasamy** [comment]
jenkins run tests please

**Comment by PriyangaRamasamy** [comment]
> > I would go with '-', as I see other directories in - in this repo.
> 
> The coding guidelines has it as '_'

done

**Review by SunnySrivastava1984 — APPROVED** [review]
Looks good to me.

**Pull request merged**